src/query/list_file.c

```diff
@@ -371,7 +370,7 @@ qfile_list_cache_cleanup (THREAD_ENTRY * thread_p)
   for (candidate_index = 0; candidate_index < bh->element_count; candidate_index++)
     {
       bh_element_at (bh, candidate_index, &candidate);
-      qfile_end_use_of_list_cache_entry (thread_p, candidate.qcache, true);
+      qfile_delete_list_cache_entry (thread_p, candidate.qcache, NULL);
```

Comment:
What is the difference between qfile_end_use_of_list_cache_entry() and qfile_delete_list_cache_entry()?
Is it safe to delete immediately?

Reply:
qfile_end_use_of_list_cache_entry: remove only the related information
qfile_delete_list_cache_entry: remove the result file from cache entry.